modules/simple_forms_api/app/form_mappings/vba_40_10007.json.erb

```diff
@@ -0,0 +1,3 @@
+{
+  "form1[0].#subform[0].TextField1[0]": "<%= data.dig('name', 'first') + ' ' + (data.dig('name', 'middle') || '') + ' ' + data.dig('name', 'last') %>"
```

Comment:
This should be:
```
"form1[0].#subform[0].TextField1[0]": "<%= data.dig('application', 'applicant', 'name', 'first') + ' ' + (data.dig('application', 'applicant', 'name', 'middle') || '') + ' ' + data.dig('application', 'applicant', 'name', 'last') %>"
```
if you're trying to get the applicant's name here. You'll need to repeat this for all fields of the PDF and check the actual PDF (after running tests) to ensure the right fields are filled in with the right contents.